### DISCUSSION TITLE
Zooz ZEN51 and ZEN52 Missing from Database

### DISCUSSION THREAD

**Discussion by vmcosco** [zwave-js/zwave-js#4281] [General] [answered]
Couple of Zooz products missing from the database:

ZOOZ 700 SERIES Z-WAVE PLUS DRY CONTACT RELAY ZEN51 - [link](https://www.thesmartesthouse.com/collections/zooz/products/zooz-700-series-z-wave-plus-dry-contact-relay-zen51)

ZOOZ 700 SERIES Z-WAVE PLUS DOUBLE RELAY ZEN52 - [link](https://www.thesmartesthouse.com/collections/zooz/products/zooz-700-series-z-wave-plus-double-relay-zen52)

**Comment by AlCalzone** [comment, marked as answer]
PR is already open: https://github.com/zwave-js/node-zwave-js/pull/4262
